doc/conf.py

```diff
@@ -274,6 +274,12 @@ def setup(app):
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {'http://docs.python.org/': None}
 
+try:
+    from mayavi import mlab
+    use_mayavi = True
+except ImportError, e:
```

Comment:
This is not python 3 compatible and is old way of doing things in python 2. Slightly annoying that the tests pass, but oh well this is part of the doc generation and not really the sphinxgallery package ...

Since you are not using the exception you might as well just do:

```python
except ImportError:
```